package/tvheadend-app-osmc/files/DEBIAN/postinst

```diff
@@ -41,6 +94,61 @@ if [ "$1" = "configure" ]; then
 		fi
 	fi
 
-	if ischroot; then exit 0; fi
+	# start tvh so that it updates any old user files
+	echo Starting tvheadend to finalise setup for osmc
 	if systemctl is-enabled tvheadend.service >/dev/null; then systemctl start tvheadend.service; fi
+	#systemctl restart tvheadend
+	sleep 5
```

Comment:
I should have gone back and branched it from the original.  I hope you are just diffing the latest commit with your master.